providers/os/connection/snapshot/partition.go

```diff
@@ -11,6 +11,7 @@ type PartitionInfo struct {
 	Name   string
 	FsType string
 
+	Aliases []string
```

Comment:
can you write a comment above on what those aliases are?

Reply:
updated them all

Reply:
nice!